app/Http/Controllers/HomeController.php

```diff
@@ -22,8 +22,25 @@ public function __construct()
      *
      * @return \Illuminate\Http\Response
      */
+
+    public function getWordsLearnedCount($lessons) 
+    {
+        $learnedWordsCount = 0;
+        foreach($lessons as $lesson) {
+            $learnedWordsCount += count($lesson->learnedWords);
+        }
+
+        return $learnedWordsCount;
+    }
+
```

Comment:
you can put this function in Lesson model, it is more direct and can be reuse by other classes every time they instantiate Lesson.